src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  align-items: center;
+
+  grid-template-columns: repeat(4, 200px);
+  grid-auto-flow: row;
+
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, 200px);
```

Comment:
Must be 2 columns between 4**88**-767. you did it will be 2 columns between 4**69**-768.